main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -179,7 +179,15 @@ object Main {
     val flix = new Flix()
     flix.setOptions(options)
     for (file <- cmdOpts.files) {
-      flix.addPath(file.toPath)
+      val ext = file.getName.split('.').last
+      ext match {
+        case "flix" => flix.addPath(file.toPath)
+        case "fpkg" => flix.addPath(file.toPath)
```

Comment:
I wonder if this should be refactored, but that could be in another PR.

Reply:
Yes, we need some file type checking utils, since there's some repeated code in a number of places.